pace/pace.d.ts

```diff
@@ -0,0 +1,113 @@
+// Type definitions for pace v0.7.5
+// Project: https://github.com/HubSpot/pace
+// Definitions by: Borislav Zhivkov <https://github.com/borislavjivkov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface PaceOptions {
```

Comment:
http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.